src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/GraphNodes/ViewProviders/GraphViewProviderBase.cs

```diff
@@ -145,7 +145,8 @@ public virtual bool ShouldTrackChanges(string projectPath, string updatedProject
         {
             nodesToRemove = existingChildren.Except(updatedChildren).ToList();
             nodesToAdd = updatedChildren.Except(existingChildren).ToList();
-            return nodesToAdd.Any() || nodesToRemove.Any();
+
+            return nodesToAdd.Count != 0 || nodesToRemove.Count != 0;
```

Comment:
I wonder if in this case keeping the `.Any()` and removing the `.ToList()` might actually be better overall? Depends on how big the incoming lists are likely to be and how often there is expected to be changes, which is info I don't have.

Reply:
The `ToList` calls support returning `IReadOnlyList<T>` via the `out` params, not just the `Any` calls.